flexy-osisoft-connector/src/com/hms/flexyosisoftconnector/OSIsoftServer.java

```diff
@@ -115,7 +115,9 @@ public int RequestHTTPS(String CnxParam, String Method, String Headers, String T
 
    // Looks up and sets a tags PI Point web id.  If the tag does not exist
    // in the data server a PI Point is created for that tag.
-   public void setTagWebId(Tag tag) throws JSONException {
+   public int setTagWebId(Tag tag) throws JSONException {

```

Comment:
can we have a type for the return value here to make it more self documenting as to what the potential return values are?